target/linux/ramips/dts/mt7621_ubnt_edgerouter-x-sfp.dts

```diff
@@ -1,4 +1,4 @@
-#include "mt7621_ubnt_edgerouter-x.dtsi"
+#include "mt7621_ubnt_e50-series.dtsi"
```

Comment:
I'd personally have omitted the "-series" here, and just called it mt7621_ubnt_e50.dtsi